recipe/meta.yaml

```
@@ -1,29 +1,32 @@
 {% set name = "opentelemetry-api" %}
-{% set version = "1.23.0" %}
-{% set sha256 = "14a766548c8dd2eb4dfc349739eb4c3893712a0daa996e5dbf945f9da665da9d" %}
+{% set version = "1.26.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opentelemetry_api-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  sha256: 2bd639e4bed5b18486fef0b5a520aaffde5a18fc225e808a1ac4df363f43a1ce
+  patches:
+    - patches/0000-fix-classifier-clash-in-pyproject.patch
 
 build:
   number: 0
   skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:
+    - patch # [not win]
+    - m2-patch  # [win]
   host:
     - pip
     - python
     - hatchling
   run:
     - deprecated >=1.2.6
-    - aiocontextvars  # [py<37]
-    - importlib-metadata >=6.0,<7.0
+    - importlib-metadata >=6.0,<=8.2.0
```

```suggestion
    - importlib-metadata >=6.0,<=8.0.0
```
https://github.com/open-telemetry/opentelemetry-python/blob/d84fc6cb9425c2f8c42d528108f651e527d81ef5/opentelemetry-api/pyproject.toml#L32
make sure to check the version-tagged commit of the code rather than the head of the main branch